src/actions/delivery.ts

```diff
@@ -18,7 +18,8 @@ const Delivery = {
   density: density,
   defaultImage: defaultImage,
   colorSpace: colorSpace,
-  colorSpaceFromICC: colorSpaceFromICC
+  colorSpaceFromICC: colorSpaceFromICC,
+  quality
```

Comment:
why is this not quality:quality?

Reply:
no real reason really, I'll adjust in future prs.